LineApp/src/main/java/org/example/utils/HibernateUtil.java

```diff
@@ -0,0 +1,32 @@
+package org.example.utils;
+
+import org.example.models.*;
+import static java.lang.Boolean.TRUE;
+import static org.hibernate.cfg.AvailableSettings.*;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = null;
+    static{
+        sessionFactory = new Configuration()
+                //add mapping class
+                .addAnnotatedClass(PullRequest.class)
+                //connect to database
+                .setProperty(URL, "jdbc:postgresql://localhost:5432/{dbname}")
```

Comment:
it's ok for now, in the future you'll need to make a couple of changes: a) extract all of that to config file b) never ever hardcode password here